provider/testdata/Cargo.toml

```diff
@@ -110,42 +90,8 @@ icu_datetime_experimental = [
 ]
 buffer = ["icu_provider_blob", "icu_provider/sync", "icu_provider_adapters/serde", "lazy_static"]
 std = []
-bin = [
-    "bytes",
-    "clap",
-    "eyre",
-    "simple_logger",
-    "log",
-    "reqwest",
-    "tokio",
-    "futures",
-    "std",
-    "metadata",
-    "icu_datagen",
-    "icu_provider_fs",
-    "zip",
-]
-# Enables programmatic access to the sources.toml file
-# If you want this to be made public stable, file an issue on the ICU4X repository.
-metadata = [
-    "icu_locid/serde",
-    "toml",
-    "serde",
-    "displaydoc",
-    "writeable",
-    "std"
-]
-all_features_hack = []
-
-[[bin]]
-name = "icu4x-testdata-download-sources"
-path = "src/bin/download-sources.rs"
-required-features = ["bin"]
-
-[[bin]]
-name = "icu4x-testdata-datagen"
-path = "src/bin/datagen.rs"
-required-features = ["bin"]
+internal_all_features_hack = []
```

Comment:
thought: we may not still need this https://github.com/frewsxcv/cargo-all-features/pull/14

Reply:
This is not for `cargo-all-features`, but for `cargo {check,clippy,build} --all-features`.

Reply:
ah, right